commands/file_operation_windows.go

```diff
@@ -0,0 +1,14 @@
+// +build windows
+
+package commands
+
+import "fmt"
+
+func createDefaultConfigFileIfNotExists() error {
```

Comment:
Where does this code get called? I mean, where is the branching logic to know that this is for windows.

Reply:
This method was part of root.go. Now moving this method to operating system specific file will make sure corresponding implementation is called without need of including checks like below:
```
if runtime.GOOS == "windows" {
			// do windows specific and so on.
		}
```

Reply:
Oh I understand now